README.md

```diff
@@ -10,19 +10,25 @@ how to work with the Pulumi resource lifecycle to accomplish randomness safely a
 
 ## Example
 
-For example, to generate a random string, simply allocate a resource:
+For example, to generate a random password, simply allocate a RandomPassword resource,
```

Comment:
> For example, to generate a random password, simply allocate a RandomPassword resource,
and then we can use its `result` output property, of type `Output<string>`, to pass
to another resource.

I would rephrase this to:
For example, to generate a random password, allocate a _RandomPassword_ resource
and then use its `result` output property (of type `Output<string>`) to pass
to another resource.